lib/config.shlib

```diff
@@ -12,9 +12,9 @@ read() { # path, key -> value
 }
 
 delete() { # path, key
-  test -f "$1" && sed -i "" -e "/^$(echo $2 | sed_escape).*$/d" "$1"
-  # UNIX VERSION:
-  # test -f "$1" && sed -i "/^$(echo $2 | sed_escape).*$/d" "$1"
+  test -f "$1" && sed -i "/^$(echo $2 | sed_escape).*$/d" "$1"
+  # MacOs
```

Comment:
https://stackoverflow.com/questions/394230/how-to-detect-the-os-from-a-bash-script

Vllt einfach abfragen, ob es macos ist?